### DISCUSSION TITLE
Unable to use tags and categories outside _posts

### DISCUSSION THREAD

**Discussion by wildone** [mmistakes/minimal-mistakes#3029] [Q&A] [answered]
## Environment

- Minimal Mistakes version: latest
- Ruby gem or remote theme version: 2.0
- Jekyll version: latest
- Git repository URL: https://github.com/aem-design/aem.design
- GitHub Pages hosted (if yes provide URL to site): https://aem.design/
- Operating system: Ubuntu 20

## Expected behavior

I expect that when tags and categories are working on `_posts`, I can add same configuration to other pages in the site and they would behave the same. 

I have tested this with liquid and jekyll-archives. I might be missing something but its not obvious to me as it works fine for `_posts`.

## Steps to reproduce the behavior

Un commenting following line in https://github.com/aem-design/aem.design/blame/master/_archives/001-davids-model.md does not result in respective categories and tags being generated.

```
#categories:
#  - archives
#tags:
#  - reference
#  - backup
```

## Other

You bloody legends!

**Comment by mmistakes** [comment, marked as answer]
Duplicate. See https://github.com/mmistakes/minimal-mistakes/issues/405#issuecomment-235148252

**Comment by wildone** [comment]
cool thanks! :D
